Ginger/Ginger/Actions/ActionConversion/SelectActivityWzardPage.xaml.cs

```diff
@@ -63,6 +63,7 @@ private void SetGridsView()
             xGrdGroups.SetAllColumnsDefaultView(defView);
             xGrdGroups.InitViewItems();
             xGrdGroups.SetTitleLightStyle = true;
+            xGrdGroups.btnMarkAll.Visibility = System.Windows.Visibility.Visible;
 
```

Comment:
please update MarkUnMarkAllActivities function code to fit my comments in the other page